### DISCUSSION TITLE
Question about the position of the object after write_root_state_to_sim

### DISCUSSION THREAD

**Discussion by johnnylu305** [isaac-sim/IsaacLab#581] [Q&A] [answered]
I tried running ./isaaclab.sh -p source/standalone/tutorials/01_assets/run_rigid_object.py. The cone's position changes randomly, but the position property on the GUI (stage) does not change. Additionally, the origin dot of the cone does not move with the cone. Could you provide some insights into this design? I am wondering if there is any way to move the origin dot along with the cone. 

![image](https://github.com/isaac-sim/IsaacLab/assets/22707984/0f3ab370-e6da-4042-859a-6b174b2f0d79)

**Comment by Mayankm96** [comment, marked as answer]
While running the simulation, we disable USD read/write operations for performance reasons (also known as using Fabric in Omniverse). Due to this, you don't see any data in the GUI getting updated. If you want to disable fabric for debugging, you can set the flag for it to False in the `SimulationCfg`:

Example:

```python
sim_cfg = SimulationCfg()
sim_cfg.use_fabric = False
sim_cfg.use_gpu_pipeline = False
sim_cfg.physx.use_gpu = False
```

**Comment by johnnylu305** [comment]
Thanks for the explanation
